Gemfile

```diff
@@ -80,12 +80,12 @@ gem 'htmltoword'
 # gem 'debugger'
 
 group :development, :test do
```

Comment:
The tests were written with limited knowledge of Rails, Rspec and Selenium, and I am certain there are many places we can improve those tests. @astrilet, could you please capture that as an issue in the backlog to reevaluate that test using Nokogiri. And it's a good point that these issues won't affect production's security. Let's merge the version bump, for now, to ease off the concerns over the gemnasium warnings, and work on fixing the tests as the next step.